### PR TITLE
New timer icon set

### PR DESCRIPTION
**Pull request by pieterdd**
I've been experimenting with some new timer icons. Here's a preview:

Screenshot 2020-09-20 135357

Screenshot 2020-09-20 152611

_(the dotted border around the coffee mug is from Gtk's button decoration, not the icon itself, it just happened to be active while I was taking the above screenshot)_

After lots of iterations, I think these look pretty well at 16x16. I steered clear from a very gradiented look and kept the icon style fairly flat. However, I still have two concerns:

- The current icon set features a shared primary color (beige) with only the last icon having a minor green accent. In the new set, on the other hand, every icon has a different primary color. This creates more color contrast and can therefore seem a bit more in-your-face. I don't know if you have a personal preference for neutral or contrasting.
- These might not work well on dark backgrounds like the [Windows applet](https://workrave.org/docs/applets/windows/). There was no Workrave toolbar on my Windows install so I haven't been able to check yet. I'm pretty sure that dark icons on dark backgrounds usually don't work well without a light border or some other trick though. Also, if you ever wish to make Workrave dark mode compatible, then this is a must have.

tl;dr I tried making some icons, it's harder than I thought, opinions about the art direction welcome.

### CODE REVIEW

**Comment by rcaelers** [comment]
Looks good! I also tried to create new icons with a flat style for the website, but ultimately gave up... (I am not experienced enough with Affinity)

I will check if it is easy to support icon themes in Workrave. This also makes it easier to experiment. 

Workrave tries to adapt itself to the selected theme (at least on Linux). So I think it should be dark mode compatible.

**Comment by pieterdd** [comment]
I'm all for icon theme support! You know, I've been using Workrave on and off for what feels like more than 10 years. It's the most feature-complete tool I know in its category, it's open source and it works on multiple platforms.

But I'm curious to see Workrave evolve on the UI/UX front, so experimenting with icon themes sounds like a great idea. That way I might be able to come up with a modern replacement for icons like the [CRT monitor](https://github.com/rcaelers/workrave/blob/branch_v1_10/frontend/common/share/images/display.png) while people who prefer things how they currently are can stick to the vintage set.

**Comment by senpl** [comment]
Icon for daily limit seems missleading. Old one was more suggestive, new one looks like power off which is not exactly the same thing.

**Comment by pieterdd** [comment]
Personally I've never actually exited the room for the rest of the day after the daily limit timer ran out 🙈 Do you keep your computer powered on after the daily limit runs out?

**Comment by senpl** [comment]
> Personally I've never actually exited the room for the rest of the day after the daily limit timer ran out

Sure not in this time of the year, but I usually try to go away as soon as I'm done for the day. And I know some miners and admins let their computer running after workday ends. I also do not like new coffie break icon, it is just not clear enough.
